valhalla/sif/dynamiccost.h

```diff
@@ -177,7 +177,7 @@ class DynamicCost {
   virtual bool AllowedReverse(const baldr::DirectedEdge* edge,
                               const EdgeLabel& pred,
                               const baldr::DirectedEdge* opp_edge,
-                              const baldr::GraphTile*& tile,
```

Comment:
i dont know why we were passing a reference to the pointer but it wasnt needed so i removed it everywhere